README.md

```diff
@@ -20,11 +20,18 @@ In order to interact with any package contained in this repository you will need
 
 - [**NVM**](https://github.com/nvm-sh/nvm/blob/master/README.md)
 - [**Homebrew**](https://brew.sh/)
-- [**Node.js@18.x.x**](https://nodejs.org/)
+- [**Node.js@16.x.x**](https://nodejs.org/) `16.x.x`
```

Comment:
This should now be node 18

Reply:
ah nice catch, just reran everything with node 18, all clear 👍